File: 251004/Klimovich/distcomp/src/main/java/com/example/distcomp/dto/LabelRequestTo.java
package com.example.distcomp.dto;

public class LabelRequestTo {
    private long id;

    private String name;

    public LabelRequestTo() {
    }

    public LabelRequestTo(long id, String name){
        this.id = id;
        this.name = name;
    }


    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: 251003/Golovin/appLab4/common/src/main/kotlin/com/example/eger/dc/lab4/dto/request/LabelRequestTo.kt
package com.example.eger.dc.lab4.dto.request

import com.example.eger.dc.lab4.bean.Label
import kotlinx.serialization.Serializable

@Serializable
data class LabelRequestTo(
	private val name: String
) {
	fun toBean(id: Long?): Label = Label(
		id, name
	)

	init {
		require(name.length in 2..32)
	}
}